frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/highlightNodesAndEdges.ts

```diff
@@ -127,9 +127,13 @@ export const highlightNodesAndEdges = (
     const sourceTableName = edge.source
     const targetTableName = edge.target
     if (!edgeMap.has(sourceTableName)) {
-      edgeMap.set(sourceTableName, [])
+      edgeMap.set(sourceTableName, new Set())
     }
-    edgeMap.get(sourceTableName)?.push(targetTableName)
+    if (!edgeMap.has(targetTableName)) {
+      edgeMap.set(targetTableName, new Set())
+    }
+    edgeMap.get(sourceTableName)?.add(targetTableName)
+    edgeMap.get(targetTableName)?.add(sourceTableName)
```

Comment:
EdgeMap has so far only been able to cache information in the direction of the child from the parent.
Therefore, it was possible to highlight a child node when hovering a parent node, but not the reverse.

Added reverse cache and also set to Set to remove duplicates due to possible duplication.